gpt_2_simple/gpt_2.py

```diff
@@ -144,7 +149,7 @@ def finetune(sess,
     See that file for parameter definitions.
     """
 
-    assert model_name not in ['774M', '1558M'], "Currently, modern GPUs cannot finetune the 774M GPT-2 model or larger."
+    assert model_name not in ['774M', '1558M'] and not multi_gpu, "Currently, a modern single GPU cannot finetune the 774M GPT-2 model or larger."
```

Comment:
s/`and not multi_gpu`/`or multi_gpu`

Reply:
Nice catch. I wrote that kind of quickly. During testing I had that assert commented out.